src/shared/constants/chart-props.js

```diff
@@ -81,3 +81,19 @@ export const barChartDefaultProps = {
   groupByKey: '',
   valueKey: '',
 }
+
+export const typographyPropTypes = {
+  typographyProps: PropTypes.shape({
+    fontFamily: PropTypes.string.isRequired,
+    fontSize: PropTypes.number.isRequired,
+    textColor: PropTypes.string.isRequired
+  })
+}
+
+export const typographyDefaultProps = {
+  typographyProps: {
+    fontFamily: ' \'Open Sans\', sans-serif',
```

Comment:
when literal quotes are needed, and in this case especially when official CSS convention is to use double quotes for when the font-family contains whitespace, it's better to just mix-in the quotes:

```js
    fontFamily: '"Open Sans", sans-serif',
```

relevant eslint rule configuration can be found [here](https://eslint.org/docs/rules/quotes#avoidescape), which [we already use](https://github.com/EQWorks/chart-system/blob/master/.eslintrc.js#L38).

Reply:
Ok, so why here it is like this: https://github.com/EQWorks/react-labs/blob/master/src/theme/typography.js ?